jenkins/dockerfiles/build-k8s/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# Container to driver Kubernetes cluster from KernelCI
+#
+FROM kernelci/build-base
+MAINTAINER "Kevin Hilman <khilmnan@baylibre.com>"
+
+#
+# Install gcloud-sdk
+# From: https://cloud.google.com/sdk/docs/downloads-apt-get
+#
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates gnupg curl
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+RUN apt-get update && apt-get install -y google-cloud-sdk kubectl
+
+#
+# Azure CLI
+# From: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest
+#
+RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor |  tee /etc/apt/trusted.gpg.d/microsoft.asc.gpg > /dev/null
+RUN AZ_REPO=buster && echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | tee /etc/apt/sources.list.d/azure-cli.list
+RUN apt-get update && apt-get install -y azure-cli
+
+#
+# Kubernetes python lib (need latest from pip)
```

Comment:
Do we really need the _latest_ version?  Having a numbered version would probably be a bit safer and also trigger image rebuilds when the number changes in the Dockerfile.

Reply:
The version deps seem buggy/incomplete in the original gcloud packages, so this was the only combination I was able to make work reliably.

Reply:
OK fine.  If this breaks one day then we'll hopefully be able to go back to released versions that do work.  Maybe we should tag the Docker images too, to be able to roll back easily.